hoomd/device.py

```diff
@@ -271,6 +272,29 @@ def is_available():
         """
         return _hoomd.isCUDAAvailable();
 
+    @contextlib.contextmanager
+    def enable_profiling(self):
+        """ Enable GPU profiling.
+
+        When using GPU profiling tools on HOOMD, select the option to disable
+        profiling on start. Initialize and run a simulation long enough that
+        all autotuners have completed, then open :py:func:`enable_profiling()`
```

Comment:
I can't remember for sure, but I don't think you need parentheses when using these kind of function references. I think it might even add them for you in the rendered docs.

Reply:
That is correct. It needs to access and inspect the object (since all Python functions are objects), so you don't need parenthesis.

Reply:
Yes, sphinx adds them in the rendered docs.